src/Cli/ExtraCommands.php

```diff
@@ -49,6 +50,138 @@ public function repoConvertData($path, $options = ['format' => 'json'])
         return $reposResult;
     }
 
+    /**
+     * @command project:update-info
+     * @param $project The github project to update info for.
+     *
+     * @return Consolidation\OutputFormatters\StructuredData\RowsOfFields
+     */
+    public function projectUpdateInfo($project, $options = [
+        'as' => 'default',
+        'codeowners' => '',
+        'support-level-badge' => '',
+        'branch-name' => 'project-update-info',
+        'commit-message' => 'Update project information.',
+        'pr-title' => 'Update project information.',
+        'pr-body' => '',
+        'base-branch' => 'master',
+    ])
+    {
+        if (count(explode('/', $project)) != 2) {
+            throw new \Exception("Invalid project name: $project");
+        }
+        if (empty($options['codeowners']) && empty($options['support-level-badge'])) {
+            throw new \Exception("Must specify at least one of --codeowners or --support-level-badge");
+        }
+        $url = "git@github.com:$project.git";
+        $remote = new Remote($url);
+        $api = $this->api($options['as']);
+        $dir = sys_get_temp_dir() . '/hubph/' . $remote->project();
+        $baseBranch = $options['base-branch'];
+        $workingCopy = WorkingCopy::cloneBranch($url, $dir, $baseBranch, $api);
+
+        // Set logger for workingCopy, is this correct?
+        if ($this->logger) {
```

Comment:
@greg-1-anderson is this ok or is there a better way? Without this, pr function was failing because of logger was null.

Reply:
Yeah, that's a fine way to inject the logger

Reply:
Thanks